api/rest/deploy.go

```diff
@@ -173,30 +177,7 @@ func setDeploy(request *restful.Request, response *restful.Response) {
 		return
 	}
 
-	var deleteFlag = false
-	for _, deployPrePlan := range deployPre.DeployPlans {
-		if deployPrePlan.PlanName == deploy.DeployPlans[0].PlanName {
-			deleteFlag = true
-		}
-	}
-
-	log.Infof("gao %+v", deploy)
-	// delete element
-	if deleteFlag {
-		for _, deployPlan := range deploy.DeployPlans {
-			for index, deployPrePlan := range deployPre.DeployPlans {
-				if deployPrePlan.PlanName == deployPlan.PlanName {
-					deployPre.DeployPlans = append(deployPre.DeployPlans[:index], deployPre.DeployPlans[index+1:]...)
-					break
-				}
-			}
-		}
-	} else {
-		log.Info("gao jinqiao ")
-		deployPre.DeployPlans = append(deployPre.DeployPlans, deploy.DeployPlans...)
-	}
-	log.Infof("gao jianqiao %+v", deployPre.DeployPlans)
-
+	deployPre.DeployPlan = deploy.DeployPlan
```

Comment:
if the uid do not match ，this is a ACL issue. Actually, this func isn't used by UI.